### PR TITLE
[zh] Sync kubelet-credential-provider.md and downward-api-volume-expose-pod-information.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@fujitsu.com>

Work for https://github.com/kubernetes/website/issues/33477 tasks-13
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

File: content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md [hunk 1]
```diff
@@ -207,8 +216,20 @@ Some example values of `matchImages` patterns are:
...
+
+## {{% heading "whatsnext" %}}
+
+<!--
+* Read the details about `CredentialProviderConfig` in the
+  [kubelet configuration API (v1alpha1) reference](/docs/reference/config-api/kubelet-config.v1alpha1/).
+* Read the [kubelet credential provider API reference (v1alpha1)](/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/).
+-->
+* 阅读 [kubelet 配置 API (v1alpha1) 参考](/zh/docs/reference/config-api/kubelet-config.v1alpha1/)中有关 `CredentialProviderConfig` 的详细信息。
+* 阅读 [kubelet 凭据提供程序 API 参考 (v1alpha1)](/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)。
```
Comment (tengqm) [inline content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:234]:
```suggestion
* 阅读 [kubelet 凭据提供程序 API 参考 (v1alpha1)](/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)。
```

File: content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 2]
```diff
@@ -244,13 +255,16 @@ default value of `1` which means cores for cpu and bytes for memory.
...
 
-查看 `downwardAPI` 下面的 `items` 数组。每个数组元素都是一个 DownwardAPIVolumeFile。
+查看 `downwardAPI` 下面的 `items` 数组。每个数组元素都是一个
+[`DownwardAPIVolumeFile`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#downwardapivolumefile-v1-core)。
 
-第一个元素指定名为 `client-container` 的容器中 `limits.cpu` 字段的值应保存在名为
-`cpu_limit` 的文件中。
+第一个元素指定在名为 `client-container` 的容器中，
+以 `1m` 所指定格式的 `limits.cpu` 字段的值应保存在名为 `cpu_limit` 的文件中。
+`divisor` 字段是可选的，默认值为 `1`，表示 cpu 的核心和内存的字节。
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md:267]: Thanks for the review, fixed as suggested.

File: content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 3]
```diff
@@ -283,39 +297,50 @@ You can use similar commands to view the `cpu_request`, `mem_limit` and
...
+
   * `metadata.name` - Pod 名称
   * `metadata.namespace` - Pod 名字空间
   * `metadata.uid` - Pod 的 UID
-  * `metadata.labels['<KEY>']` - Pod 标签 `<KEY>` 的值 (例如, `metadata.labels['mylabel']`）
-  * `metadata.annotations['<KEY>']` - Pod 的注解 `<KEY>` 的值（例如, `metadata.annotations['myannotation']`）
+  * `metadata.labels['<KEY>']` - Pod 标签 `<KEY>` 的值
+    （例如, `metadata.labels['mylabel']`）
+  * `metadata.annotations['<KEY>']` - Pod 的注解 `<KEY>` 的值
+    （例如, `metadata.annotations['myannotation']`）
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md:331]: Thanks, done.

File: content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 4]
```diff
@@ -244,13 +255,16 @@ default value of `1` which means cores for cpu and bytes for memory.
 
 Create the Pod:
 -->
-在这个配置文件中，你可以看到 Pod 有一个 `downwardAPI` 类型的卷，并且挂载到容器的
-`/etc/podinfo` 目录。
+在这个配置文件中，你可以看到 Pod 有一个
+[`downwardAPI` 卷](/concepts/storage/volumes/#downwardapi)，
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md:259]: Thanks for the review, added '**/zh**' as suggested.

But I found that the en page is wrong, it should be:
(**/docs**/concepts/storage/volumes/#downwardapi)

I will send a PR to add '**/docs**' in the en page and then update this page again to keep it in sync.

File: content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md (hunk 1 above)
Comment (fenggw-fnst) [reply content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:234]: Thanks, but the zh page for kubelet-credentialprovider.v1alpha1.md does not exist at present, so I recommend linking it to the en page for now. WDYT?

File: content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 5]
```diff
@@ -7,12 +7,15 @@ weight: 40
...
+[`DownwardAPIVolumeFile`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#downwardapivolumefile-v1-core)
+to expose information about itself to Containers running in the Pod.
+A `DownwardAPIVolumeFile` can expose Pod fields and Container fields.
 -->
-此页面描述 Pod 如何使用 DownwardAPIVolumeFile 把自己的信息呈现给 Pod 中运行的容器。
-DownwardAPIVolumeFile 可以呈现 Pod 的字段和容器字段。
+此页面描述 Pod 如何使用
+[`DownwardAPIVolumeFile`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#downwardapivolumefile-v1-core)
+把自己的信息呈现给 Pod 中运行的容器。
+`DownwardAPIVolumeFile` 可以呈现 Pod 的字段和容器字段。
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md:18]: Thanks for the review, updated as suggested.

File: content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: Sent #33933 to fix the en page, PTAL, thanks.

[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: #33933 has been merged, so update zh page as well.

File: content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:234]: At this stage, there are many dangling links to be fixed (both in the En directory and the Zh directory). We can worry about that later.

Comment (fenggw-fnst) [reply content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:234]: Understood, thanks.

**Pull request merged**